lib/rule-stub.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @fileoverview Provides a rule stub for missing or deprecated rules.
+ * @author Ian VanSchooten
+ */
+
+"use strict";
+
+/**
+ * Provide a stub rule with a given message
+ * @param  {string} message The message to be displayed for the rule
+ * @returns {Function}      Stub rule module
+ */
+function stub(message) {
+    if (message) {
+        /* eslint-disable func-style */
```

Comment:
I was having trouble with getting `no-inner-declarations`, but I've figured it out now.  Thanks.